accounts/urls.py

```diff
@@ -1,10 +1,15 @@
-from django.conf.urls import patterns, url
+from django.conf.urls import patterns, url, include
 from django.views.generic import TemplateView
+#from social_auth.views import
 
 from accounts import views
 
 urlpatterns = patterns('',
-    url(r'^$', views.NewUserView.as_view(), name='accounts'),
+    url(r'^$', views.NewUserView.as_view(), name='register'),
     url(r'^success/$', TemplateView.as_view(template_name="accounts/success.html"), name='success_created'),
-    url(r'^activate/(?P<activation_key>\w+)/$', views.activate)
+    url(r'^activate/(?P<activation_key>\w+)/$', views.activate),
+    url('', include('social.apps.django_app.urls', namespace='social')),
```

Comment:
To jest OK, to nie pasuje do pustego ciągu znaków tylko do dowolnego, jednocześnie nie "konsumując" nic, więc cały url jest dopasowywany potem do tego w urlach z social'a.